pkg/arvo/gen/group-store/remove.hoon

```diff
@@ -0,0 +1,10 @@
+::  group-store|remove: remove members from a group
+::
+/-  *group-store
+:-  %say
+|=  $:  [now=@da eny=@uvJ =beak]
+        [[=path members=(list ship) ~] ~]
+    ==
+:-  %group-action
```

Comment:
great additions! love these generators. Could you put them under the folder `groups` instead of `group-store`? It would give an interface akin to the publish interface you put forward. Thanks!

Reply:
I would if I could, but `%say` generators send their output as a poke to the app based on the path they're in. I can't put it into `groups` because there is no `/app/groups` to catch the resulting poke.

Reply:
`:foo|bar` is short for `:foo +foo/bar`, so you could, but it'd be awkward.